Documentation/configuration/argocd-issues.rst

```diff
@@ -99,3 +99,29 @@ of Argo CD related issues on GitHub
 <https://github.com/cilium/cilium/issues?q=is%3Aissue+argocd>`__.
 If you can't find an issue that relates to yours, create one and/or seek help
 on `Cilium Slack`_.
+
+Application chart for Cilium deployed to Talos Linux fails with: field not declared in schema
+=============================================================================================
+
+When deploying Cilium to Talos Linux with ArgoCD, some users have reported issues due to Talos Security configuration. ArgoCD may fail to deploy the application with the message:
+
+.. code-block:: shell
+    Failed to compare desired state to live state: failed to calculate diff: 
+    error calculating structured merge diff: error building typed value from live 
+    resource: .spec.template.spec.securityContext.appArmorProfile: field not 
+    declared in schema
+
+Solution
+--------
+
+Set the Application to `ServerSideApply=true``, visit the `ArgoCD documentation <https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#server-side-apply>`__ for further details. 
+
+.. code-block:: yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    spec:
+      syncPolicy:
+        syncOptions:
+        - ServerSideApply=true
+
+
```

Comment:
This is not precise, users can't tell where to set the option without looking at the example.

I think you need to skip a line between the `code-block` directive and its content (and I think this is because we pass an argument to the directive; otherwise, it wouldn't be mandatory).

```suggestion
Add option ``ServerSideApply=true`` to list ``syncPolicy.syncOptions`` for the Application.

.. code-block:: yaml

    apiVersion: argoproj.io/v1alpha1
    kind: Application
    spec:
      syncPolicy:
        syncOptions:
        - ServerSideApply=true

Visit the `ArgoCD documentation <https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#server-side-apply>`__ for further details. 
```